FinSearchUnified/Tests/Bundle/ProductNumberSearchTest.php

```diff
@@ -264,6 +269,96 @@ public function testProductNumberSearchResultWithAllFilters(SimpleXMLElement $xm
         }
     }
 
+    public function selectedFiltersProvider()
+    {
+        $xmlResponse = $this->getXmlResponse();
+        $filters = $xmlResponse->addChild('filters');
+
+        $this->setPriceFilter($filters);
+
+        return [
```

Comment:
Can we broaden this a little bit? Right now only the price filter is tested. Let's also make sure that the other facet results are tested as well.